### DISCUSSION TITLE
[Feature] Make "create playlist"-button available in user's "My playlist" section

### DISCUSSION THREAD

**Discussion by gtntdev** [mediacms-io/mediacms#611] [Feature Requests]
The "playlist create" button isn't very obvious (down the video player, me and a friend had to search quite some time), maybe it should be displayed in the playlist sectiont (user's "My playlist") from the menue too.

**Comment by mgogoulos** [comment]
"Create playlist" or "add to playlist" functionality need be exposed on multiple ways, and this could be one of these ways.
